src/WingetCreateCLI/Program.cs

```diff
@@ -95,48 +88,20 @@ private static void DisplayHelp(NotParsed<object> result)
                 e => e,
                 verbsIndex: true);
             Console.WriteLine(helpText);
-            Console.WriteLine();
-
-            foreach (var error in result.Errors)
-            {
-                if (error is SetValueExceptionError sve)
-                {
-                    Utils.WriteLineColored(ConsoleColor.Red, $"{sve.NameInfo.LongName}: {sve.Exception.Message}");
-                    if (sve.Exception.InnerException != null)
-                    {
-                        Utils.WriteLineColored(ConsoleColor.Red, $"{sve.Exception.InnerException.Message}");
-                    }
-
-                    if (sve.Value is IEnumerable<object> list)
-                    {
-                        foreach (var val in list)
-                        {
-                            Utils.WriteLineColored(ConsoleColor.Red, $"\t{val}");
-                        }
-                    }
-                    else
-                    {
-                        Utils.WriteLineColored(ConsoleColor.Red, $"\t{sve.Value}");
-                    }
-                }
-                else if (error is UnknownOptionError uoe)
-                {
-                    if (uoe.Token == "?")
-                    {
-                        break;
-                    }
-
-                    Utils.WriteLineColored(ConsoleColor.Red, $"Unknown option: {uoe.Token}");
-                }
-                else if (error is NoVerbSelectedError || error is HelpRequestedError)
-                {
-                    continue;
-                }
-                else
-                {
-                    Utils.WriteLineColored(ConsoleColor.Red, $"Command line parsing error: {error.Tag}");
-                }
-            }
-        }
-    }
+            Console.WriteLine();
+        }
+
+        private static void DisplayParsingErrors<T>(ParserResult<T> result)

```

Comment:
Wow, this sure looks a lot cleaner! Looks like excList is never used though?

Reply:
removed excList